File: C,C++/p11636.c
#include <stdio.h>

int main () {
    int casee = 1;

    while(1) {
        int num;
        scanf("%d", &num);
        if (num < 0) break;

        int temp = 1, res = 0;
        
        while(temp<num) {
            temp *= 2;
            res++;
        }

        printf("Case %d: %d\n", casee, res);

        casee++;
    }

    return 0;
}

File: Java/p11636.java
import java.util.Scanner;

public class p11636 {
	public static void main (String [] args) {
		Scanner input = new Scanner(System.in);
		
		int casee = 1;
		
		while (true) {
			int num = input.nextInt();
			
			if (num < 0) break;
			
			int temp = 1, res = 0;
			
			while (temp < num) {
				temp *= 2;
				res++;
			}
			
			System.out.printf("Case %d: %d\n", casee, res);
			
			casee++;
		}
	}
}